docs/extending-documentation.md

```diff
@@ -1,119 +1,169 @@
+
+```sh
+npm i @lion/button --save
 ```
 
-> Later in this blog in the `Adjust Documentation` section we will explain about why we exclude \*.override.md files here, since they are used for specific overrides and not to be included as independent story files. This file name syntax is configurable.
+Within the folder `src`, open the following file:
 
-So here we only extend the documentation of `LionTabs`, for our own `LeaTabs`. But our repository is a monorepository with multiple other packages that don't use `Lion`, which we want to include as well, so we also add the second line to include those.
+```sh
+example-button/src/ExampleButton.js
+```
 
-This step alone should already give you the `LionTabs` docs inside your own Storybook, but not yet transformed to your `LeaTabs` extension of it, which we will explain next.
+Replace the content with the following: 
 
-## Replacing imports and tagnames
+```js
+import { css } from 'lit-element';
+import { LionButton } from '@lion/button';
+
+export class ExampleButton extends LionButton {
+  static get styles() {
+    return [
+      super.styles,
+      css`
+        /* your styles can go here */
+      `
+    ];
+  }
 
-### Analysing paths
+  connectedCallback() {
+    super.connectedCallback();
+    this._setupFeature();
+  }
 
-Potentially the hardest part is to analyse your extension `LeaTabs`, and to figure out how we should transform the import paths for `LionTabs` to new paths to your `LeaTabs`.
+  _setupFeature() {
+    // your code can go here
+  }
+}
+```
 
-To do this we make use of [Providence](https://lion-web-components.netlify.app/?path=/docs/tools-providence-main--run-providence). This tool has a command that creates a full map of all the import paths of a reference project (`Lion`) and can replace them with the correct paths of a target project (`Lea`).
+You have now extended `<lion-button>` and created `<example-button>` from it. The component can be experienced in the browser by running `npm run storybook` inside the `example-button`-folder. 
+Feel free to add styles and make it your own. This can be the start of a whole set of Web Components for your Design System.
 
-So lets install it:
+For this article we assume you set up the project like mentioned before, using [Prebuilt Storybook](https://open-wc.org/demoing/) with [MDJS](https://open-wc.org/mdjs/). If you already have a repository, you can also add Storybook using [open-wc](https://open-wc.org). Enter the following:
 
 ```sh
-yarn add providence-analytics --dev
+npm init @open-wc
 ```
 
-And to use it, let's add a script to your projects `package.json`:
+And pick 'upgrade an existing project'. Or install it manually by entering the following:
 
-```json
-"scripts": {
-  "providence:extend": "providence extend-docs -r 'node_modules/@lion/*' --prefix-from lion --prefix-to lea"
-}
+```sh
+npm i @open-wc/demoing-storybook --save-dev
+```
+
+## Make a selection
+
+We need to specify which stories to load in `.storybook/main.js`.
```

Comment:
I feel like this could use some more context, selection of what? why are we making a selection? what does storybook even do?